index.html

```diff
@@ -0,0 +1,81 @@
+        </figcaption>
+      </figure>
+      <section id="tribute-info">
+        <h3>All this fast equipment save time an our energy</h3>
+        <ul>
+          <li>
+            robot, any automatically operated machine that replaces human effort, though it may not
+            resemble human beings in appearance or perform functions in a humanlike manner. By
+            extension, robotics is the engineering discipline dealing with the design, construction,
+            and operation of robots.
+          </li>
+          <li>
+            The concept of artificial humans predates recorded history (see automaton), but the
+            modern term robot derives from the Czech word robota (“forced labour” or “serf”), used
+            in Karel Čapek’s play R.U.R. (1920). The play’s robots were manufactured humans,
+            heartlessly exploited by factory owners until they revolted and ultimately destroyed
+            humanity. Whether they were biological, like the monster in Mary Shelley’s Frankenstein
+            (1818), or mechanical was not specified, but the mechanical alternative inspired
+            generations of inventors to build electrical humanoids.
+          </li>
+          <li>
+            It is can be use any place in the house exept sitting , bed room , toilet and if some
+            one is willing to put it in the toilet or bedroom or sitting room he/she is free and
+            that is why is the fastest and the best
+          </li>
+          <li>
+            Gadgets and Technology: Fact or Fiction? Is virtual reality only used in toys? Have
+            robots ever been used in battle? From computer keyboards to flash memory, learn about
+            gadgets and technology in this quiz. Learn about Isaac Asimov's Three Laws of Robotics
+            Learn about Isaac Asimov's Three Laws of Robotics A discussion of Isaac Asimov's Three
+            Laws of Robotics. © World Science Festival (A Britannica Publishing Partner) See all
+            videos for this article
+          </li>
+          <li>
+            The word robotics first appeared in Isaac Asimov’s science-fiction story Runaround
+            (1942). Along with Asimov’s later robot stories, it set a new standard of plausibility
+            about the likely difficulty of developing intelligent robots and the technical and
+            social problems that might result. Runaround also contained Asimov’s famous Three Laws
+            of Robotics
+          </li>
+        </ul>
+        <blockquote>
+          The word robotics first appeared in Isaac Asimov’s science-fiction story Runaround (1942).
+          Along with Asimov’s later robot stories, it set a new standard of plausibility about the
+          likely difficulty of developing intelligent robots and the technical and social problems
+          that might result. Runaround also contained Asimov’s famous Three Laws of Robotics
```

Comment:
Let's make the `blockquote`  element have a `cite` attribute and `cite` element. see screenshot below:

![image](https://user-images.githubusercontent.com/36057474/147682715-1a69dee3-3fa2-4383-942f-e1d8924ca160.png)

Please note that the blockquote content should be wrapped inside a `p` element.